### PR TITLE
Footer Layout Improved on Tablet and Mobile Viewport

### LINKED ISSUE #221: Footer too long in mobile view.
**Describe the bug**
Size of the footer is too much in mobile view

**To Reproduce**
1. Go to http://www.ignitus.org in mobile view

**Expected behavior**
Only limited information should be present in the footer to make it look better.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Chrome

### PR DESCRIPTION
**Pull request by lazycipher**
## Description
Added class `col-6` to make footer columns good on Mobile and Tablet.
Added  `<hr className="clearfix w-100 d-md-none" />` to make space between social icons and menu row.

## Motivation and Context
Earlier the footer was too long and was looking quite empty. It's reduced now and looks good.
Fixed #221  

## How Has This Been Tested?
I opened the website in all viewports.

## Screenshots (if appropriate):
On Desktop
On Desktop

On Mobile
On Mobile

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by Paarmita** [comment]
@plxity Have a look here- https://github.com/Ignitus/Ignitus-Client-Side-Development/pull/236
There will be no change in the year.

**Comment by lazycipher** [comment]
> Looks good for IPad view.
> @lazycipher For the mobile view, we could just list the titles of the columns and on click it, list of sections appear.
> 
> footer mobile
> 
> footer

@Paarmita, Sure, I'll work on this!


> Make one more change, instead of 2018 change the year to 2019 before copyright.

@plxity, Yes sure, I'll use some function instead of a static data to make it detect current year.

**Comment by lazycipher** [comment]
> @plxity Have a look here- #236
> There will be no change in the year.

@Paarmita, Or we can use something like `Copyright 2018 - Current Year`

**Comment by Paarmita** [comment]
@lazycipher
`Copyright 2018 - Current Year` this would mean that we have a new release every year. So this should not be done.

**Comment by Paarmita** [comment]
@lazycipher Are you working on the changes?

**Comment by lazycipher** [comment]
> @lazycipher Are you working on the changes?

I was having issues with `npm test --updateSnapshot`
I'll look into it and update asap.

**Comment by lazycipher** [comment]
Hi @Paarmita,
I updated the snapshot but now codecov check fails.
Can you guide me a bit to solve this, please?

File: src/ignitus-Footer/Styles/style.scss [hunk 1]
```diff
@@ -38,3 +38,34 @@
 #footer-bg-id {
 	margin-top: 0 !important;
 }
+
+@media only screen and (max-width : 767px) {
+	.nav-accordion {
+		.navigation {
+			h5 {
+				cursor: pointer;
+				&:after {
```
[the remark this answers is not in the document]
Comment (lazycipher) [reply src/ignitus-Footer/Styles/style.scss]: @Paarmita I've fixed this. You can check now :)

**Pull request merged**